functions.sh

```diff
@@ -19,6 +19,9 @@ function get_arch() {
 	s390x)
 		arch="s390x"
 		;;
+	arm64v8)
```

Comment:
@tianon just showed us the answer from his [comments](https://github.com/nodejs/docker-node/pull/489#discussion_r131452353) above, let me paste it here:
```
root@rpi3:~# uname -m
aarch64
```